src/modules/claim/claim.types.ts

```diff
@@ -1,4 +1,10 @@
-export const NATS_EXCHANGE_TOPIC = 'claim.exchange';
+export const NATS_EXCHANGE_TOPIC = 'CLAIM_TOPIC';
+
+export enum ClaimRequestType {
+  CLAIM_ISSUANCE = 'CLAIM_ISSUANCE',
+  CLAIM_REJECTION = 'CLAIM_REJECTION',
+  CLAIM_ENROLLMENT_REQUEST = 'CLAIM_ENROLLMENT_REQUEST',
```

Comment:
I would simplify this. Or I would change it to `ROLE_ENROLMENT_REQUEST`. I think the word `ENROLMENT` only makes sense in the context of roles
```suggestion
  CLAIM_REQUEST = 'CLAIM_REQUEST',
```